code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -134,14 +134,30 @@
 	SEND_SIGNAL(src, COMSIG_XENOMORPH_BRUTE_DAMAGE, amount, amount_mod)
 	for(var/i in amount_mod)
 		amount -= i
-
+	
```

Comment:
Not sure what this is, probably added a tab or something, which should've been there from the start.